Header header
int32 class_id
float32 presence_prob
int32 cx
int32 cy
float32 u
float32 v
float32 confidence
